File: LewisJassy/Reverse_Integer.py
def reverse_integer(n):
    # convert integer to string and reverse it
    reversed_str = str(abs(n))[::-1]
    # convert back to integer
    reversed_int = int(reversed_str)

    # restore the negative sign if needed
    if n < 0:
        reversed_int = -reversed_int
    return reversed_int

File: RyanKaleliGabriel/reverse_integer.js
function reverseInteger(num) {
    let reversedNum = 0;
    let isNegative = num < 0;
  
    // Convert negative numbers to positive for the reversal process
    num = Math.abs(num);
  
    // Reverse the digits
    while (num > 0) {
      const lastDigit = num % 10;
      reversedNum = reversedNum * 10 + lastDigit;
      num = Math.floor(num / 10);
    }
  
    // Restore the negative sign if the input number was negative
    return isNegative ? -reversedNum : reversedNum;
  }
  
  // Example usage:
  const input = parseInt(-1658, 10);
  console.log(input)
  console.log(`Reversed number: ${reverseInteger(input)}`);